frontend/src/utils/getTokenPreviewByTitle.ts

```diff
@@ -0,0 +1,6 @@
+import { tokenPreviews } from '@/tokens'
+import { TokenPreview, TokenTitle } from '@/types'
+
+export const getTokenPreviewByTitle = (title: TokenTitle): TokenPreview => {
+  return tokenPreviews.find((tokenPreview) => tokenPreview.title === title)!
```

Comment:
We could change the logic to have record instead of list of supported token previous and use satisfies to narrow down the types, I believe that after that it could be more safely to use functions:
```ts

type SupportedToken = 'SVT' | 'GLD'

export const tokenPreviews = {
  SVT: {
    title: 'SVT',
    address: import.meta.env.VITE_SIMPLE_VAULT_TOKEN_ADDRESS,
    abi: svtABI,
    Icon: TokenIcon,
    underlyingAssetTitle: 'GLD',
  },
  GLD: {
    title: 'GLD',
    address: import.meta.env.VITE_GOLD_TOKEN_ADDRESS,
    abi: gldABI,
    Icon: GoldIcon,
  },
} satisfies Record<SupportedToken, TokenPreview>
```